src/main/java/build/buildfarm/worker/UploadManifest.java

```diff
@@ -45,7 +46,10 @@ public class UploadManifest {
   private final FileStore fileStore;
   private final boolean allowSymlinks;
   private final int inlineContentLimit;
-  private final Map<Digest, Path> digestToFile;
+
+  /** -- GETTER -- Map of digests to file paths to upload. */
```

Comment:
interesting comment add - maybe drop it, or remove GETTER prefix

Reply:
Oops - part of the intelliJ refactoring. I'll remove the prefix.